torch_xla/core/xla_model.py

```diff
@@ -161,6 +161,12 @@ def xla_real_devices(devices):
     real_devices.append(device)
   return real_devices
 
+def xla_real_device_type(device):
```

Comment:
Let do:
```Python
def xla_device_hw(device):
  real_device = xla_real_devices([device])[0]
  return real_device.split(':')[0]
```

Also, can you add the proper documentation (check other functions for the style).
You make need to run `google-pyformat` over it (as well as the other python file).